### ISSUE TITLE
Assertion failure on --stop command

### ISSUE THREAD

**Issue by nickaleks** [aras-p/ClangBuildAnalyzer#26]
Command:
```~/ClangBuildAnalyzer/build/ClangBuildAnalyzer --stop ~/build-Debug profile_result.json```

Result:
```
ERROR: String ".PropertyMover1DTo3DDataFlowTest" too long to copy on line 231 in file src/external/cute_files.h (max length of 32).
ClangBuildAnalyzer: src/external/cute_files.h:215: int cf_safe_strcpy_internal(char *, const char *, int, int, const char *, int): Assertion `0' failed.
./profile_compilation.sh: line 4: 165782 Aborted                 (core dumped) ~/ClangBuildAnalyzer/build/ClangBuildAnalyzer --stop ~/build-Debug profile_result.json
```

ubuntu 18.04 with clang9

**Comment by nickaleks** [comment]
After looking at cute_files source, it appears that it limits file extension size to 32 characters set by a macro. This is a strange limitation.

**Comment by aras-p** [comment]
Yeah strange limitation indeed. Should be fixed by #27 (or at least the max extension length is bumped up).
